src/map/skill.hpp

```diff
@@ -442,6 +442,13 @@ enum e_skill_blown	{
 	BLOWN_TARGET_BASILICA		= 0x20, // If target is in Basilica area
 };
 
+// Enum for skill_dance_overlap*
+enum e_dance_overlap {
+	OVERLAP_REMOVE = 0, // Skill unit is about to be removed, remove overlap marker from overlapping units on the cell if applicable
+	OVERLAP_SET = 1, // Skill unit was placed, add overlap marker to overlapping units on the cell
+	OVERLAP_COUNT = 2, // Don't change overlap marker, just count units overlapping with skill unit (excluding)
```

Comment:
```suggestion
	OVERLAP_COUNT, // Don't change overlap marker, just count units overlapping with skill unit (excluding)
```

Reply:
Excluding what?

Reply:
If you put (excluding) behind something, it refers to excluding counting that (the word before). In this case, it excludes the skill unit for which the overlapping is checked. So if you have 3 skill units overlapping, it will return 2 because 2 other skill units over overlapping with this.